libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -435,6 +435,18 @@ class Strings {
         assertPrints(matchDetails(inputString, toFind, 10), "Searching for 'ever' in 'Never ever give up' starting at position 10: Not found")
     }
 
+    @Sample
+    fun find() {
+        val text = "k1o2t3l4i5n6"
```

Comment:
Consider using a slightly more readable string, like `"kotlin 2.1.0"`.
While it won't affect correctness, it would be much easier to understand which characters are digits and which are letters (distinguishing `l` from `1` might be tricky depending on a type font).